models/curvefi/avalanche_c/curvefi_avalanche_c_trades.sql

```diff
@@ -5,10 +5,6 @@
     file_format = 'delta',
     incremental_strategy = 'merge',
     unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address'],
-    post_hook='{{ expose_spells(\'["avalanche_c"]\',
```

Comment:
curious the intention to remove all three underlying models from the data explorer? is this a standard we should consider universally?

Reply:
context on removing it https://duneanalytics.slack.com/archives/C048R9QDWGP/p1674698430695729?thread_ts=1674696486.365059&cid=C048R9QDWGP

basically explorer shows duplicates for top level and chain level and its confusing right now
![image](https://user-images.githubusercontent.com/47720952/214923618-43a14167-97b2-4b58-812e-62cd45effb22.png)

Reply:
this deserves larger discussion. also the issue raised with 0x spells today on the explorer. we should start an initiative to outline how it currently works, how we envision it, etc.
cc @augustog @akhila-raju 

happy to remove these here -- then we need to think about how users know if these tables even exist, or do they care?
i.e. `curvefi_ethereum.trades` vs. `curvefi.trades` with a where clause for blockchain to pull subsets. do we have metrics on how often use the underlying tables? is it non-existent?

Reply:
I think anytime we have a cross-chain level table for a project, the lower levels should roll into it. This helps a lot with cleaning up the explorer. 

For example for sudoswap, since it's only ethereum then we could expose the ethereum spell hook. But if sudoswap went multichain, then we would want to only expose the multichain hook. 

This goes a long way for cross-chain standardization as well. If we had exposed a view_pools model at the cross-chain level then I think @MSilb7 would have been more likely to standardize his pools file to fit into the schema as well.

Reply:
agree that we should maybe scope out a document (and add to the readme) a proper hierarchy of when the expose hook should be used!

Reply:
@andrewhong5297 @jeff-dude would you mind documenting where the explorer could be improved? Jose is working on some changes including designing the spell tiers that @augustog asked for, so these improvements would fit in nicely with what he's already working on

Reply:
everything rolling up to cross-chain pools on the UI would probably be best. I'd use the optimism-specific view is if there's a notable performance difference, but otherwise filtering by chain is fine.

I wonder if it's worth some kind of tooltip/reminder to filter by chain though. I imagine forgetting to filter by chain will be a common error.

Reply:
But I'll always go rogue even if there's a structure :)

Reply:
we'll continue this internally, but agreed @MSilb7 -- the users building spells will know the blockchain-specific tables exist, they just may not be seen on explorer. more to come!